osu.Game.Rulesets.Mania/Skinning/Argon/ManiaArgonSkinTransformer.cs

```diff
@@ -27,6 +27,10 @@ public ManiaArgonSkinTransformer(ISkin skin, IBeatmap beatmap)
             switch (lookup)
             {
                 case GameplaySkinComponentLookup<HitResult> resultComponent:
+                    // This should eventually be moved to a skin setting, when supported.
+                    if (Skin is ArgonProSkin && resultComponent.Component >= HitResult.Great)
```

Comment:
Just to confirm intent - did this mean to use `HitResult.Great` in mania specifically (as opposed to `HitResult.Perfect`)?

Reply:
Will need player feedback on this, but `Great` is still generally considered "high enough" to not be shown it, I believe. At least in stable they don't have a detrimental effect on your score.

Reply:
Now that lazer is using ScoreV2 perhaps this should be reevaluated, as Great now reduces accuracy in mania compared to Perfect.